tests/unit/SpecHelper.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+const iframes = [];
+
+/**
+ * Create a doc for unit tests.
+ * @param {string[]} plugins Paths of plugins to load and run. Example: `/src/core/algorithms.js`
+ * @param {object} config JSON-serializable respecConfig object.
+ * @return {Promise<Document>}
+ */
+export function makePluginDoc(
+  plugins,
+  config = {},
+  { head = `<meta charset="UTF-8" />`, body = "" } = {}
```

Comment:
Is there any case where meta charset should be overriden? We should probably keep it inline below and just ="" this default.

Reply:
the check-charset test is the only reason for its existence. Maybe can use null to override it, and undefined to use default?